listeners/assistant/assistant.py

```diff
@@ -83,23 +84,27 @@ def start_assistant_thread(
 # This listener is invoked when the human user sends a reply in the assistant thread
 @assistant.user_message
 def respond_in_assistant_thread(
-    payload: dict,
-    logger: logging.Logger,
-    context: BoltContext,
     client: WebClient,
+    context: BoltContext,
+    logger: logging.Logger,
+    payload: dict,
     say: Say,
+    set_status: SetStatus,
 ):
     try:
         channel_id = payload["channel"]
         thread_ts = payload["thread_ts"]
 
-        loading_messages = [
-            "Teaching the hamsters to type faster…",
-            "Untangling the internet cables…",
-            "Consulting the office goldfish…",
-            "Polishing up the response just for you…",
-            "Convincing the AI to stop overthinking…",
-        ]
+        set_status(
+            status="Drafting...",
+            loading_messages=[
+                "Teaching the hamsters to type faster…",
+                "Untangling the internet cables…",
+                "Consulting the office goldfish…",
+                "Polishing up the response just for you…",
+                "Convincing the AI to stop overthinking…",
+            ],
+        )
```

Comment:
note: Confirmed with manual testing that all of the `loading_messages` are cycled.